third_party/CMakeLists.txt

```diff
@@ -26,5 +26,5 @@ if(MIMALLOC_STATIC)
   # (They can still be enabled via MIMALLOC_VERBOSE=1 wasm-opt ...)
   add_compile_definitions(MI_DEBUG=0)
   
-  add_subdirectory(mimalloc)
+  add_subdirectory(mimalloc EXCLUDE_FROM_ALL)
```

Comment:
From briefly looking up `EXCLUDE_FROM_ALL`, it seems that it means that this directory won't be built by default? Is that what we want, or is there a more specific way to exclude it from being installed?

Reply:
Right, I think what's happening here is that the `target_link_libraries(binaryen mimalloc-static)` causes a dependence from libbinaryen on libmimalloc-static which causes it to be built when libbinaryen does. And then the tools link it in, but it doesn't get installed, which is exactly what we want.

Reply:
I couldn't really find another way to prevent the installation (see also https://discourse.cmake.org/t/hot-to-make-add-subdirectory-prevent-install-but-still-build-as-part-of-all/8698)